patroni/ctl.py

```diff
@@ -1088,9 +1089,16 @@ def invoke_editor(before_editing, cluster_name):
     :param before_editing: human representation before editing
     :returns tuple of human readable and parsed datastructure after changes
     """
-    editor_cmd = os.environ.get('EDITOR')
+    if 'EDITOR' in os.environ:
+        editor_cmd = os.environ.get('EDITOR')
+    else:
+        for editor in ('editor', 'vi'):
+            editor_cmd = find_executable(editor)
+            if editor_cmd:
+                logging.debug('Setting fallback editor_cmd=%s', editor)
+                break
     if not editor_cmd:
-        raise PatroniCtlException('EDITOR environment variable is not set')
+        raise PatroniCtlException('EDITOR environment variable is not set. editor or vi are not available')
```

Comment:
"editor" could be replaced with editor in message to hint at wrong editor value?

Reply:
"editor" is more or less a standard thing:
```
$ ls -al /usr/bin/editor                                                                                                                                                                                                                                
lrwxrwxrwx 1 root root 24 Aug  8  2018 /usr/bin/editor -> /etc/alternatives/editor

$ ls -al /etc/alternatives/editor
lrwxrwxrwx 1 root root 9 Aug  8  2018 /etc/alternatives/editor -> /bin/nano

$ ls -al /bin/nano 
-rwxr-xr-x 1 root root 245872 Mär  6  2018 /bin/nano
```